### PR TITLE
[training_utils] fix: Update metrics.py to change calculate_log_prob_diff to calculate_diff

### PR DESCRIPTION
**Pull request by victordion**
### What does this PR do?

The function name `calculate_log_prob_diff` is misleading as the tensor it processes is not log probs but probs.

### Checklist Before Starting

- [x] Search for similar PRs. Paste at least one query link here: ...
- [x] Format the PR title as `[{modules}] {type}: {description}` (This will be checked by the CI)
  - `{modules}` include `fsdp`, `megatron`, `veomni`, `sglang`, `vllm`, `rollout`, `trainer`, `ci`, `training_utils`, `recipe`, `hardware`, `deployment`, `ray`, `worker`, `single_controller`, `misc`, `perf`, `model`, `algo`, `env`, `tool`, `ckpt`, `doc`, `data`, `cfg`, `reward`, `fully_async`, `one_step_off`
  - If this PR involves multiple modules, separate them with `,` like `[megatron, fsdp, doc]`
  - `{type}` is in `feat`, `fix`, `refactor`, `chore`, `test`
  - If this PR breaks any API (CLI arguments, config, function signature, etc.), add `[BREAKING]` to the beginning of the title.
  - Example: `[BREAKING][fsdp, megatron] feat: dynamic batching`

### Test

N/A

### API and Usage Example

N/A

### Design & Code Changes

N/A

### CODE REVIEW

File: verl/utils/debug/metrics.py [hunk 1]
```diff
@@ -55,8 +55,8 @@ def pearson_correlation_coefficient(tensor1: torch.Tensor, tensor2: torch.Tensor
     return result[0][1].detach().item()
 
 
-def calculate_log_prob_diff(log_probs1: torch.Tensor, log_probs2: torch.Tensor, mask: torch.Tensor) -> torch.Tensor:
-    full_diff = torch.abs(log_probs1 - log_probs2)
+def calculate_diff(probs1: torch.Tensor, probs2: torch.Tensor, mask: torch.Tensor) -> torch.Tensor:
+    full_diff = torch.abs(probs1 - probs2)
```
[the remark this answers is not in the document]
Comment (victordion) [reply verl/utils/debug/metrics.py:59]: This PR recognizes the current logic which is reporting on probs inseread of log probs.

File: verl/utils/debug/metrics.py [hunk 2]
```diff
@@ -111,7 +111,7 @@ def calculate_debug_metrics(data: DataProto) -> dict:
         }
 
     pearson_corrcoef = pearson_correlation_coefficient(actor_probs, rollout_probs, response_mask_bool)
-    rollout_probs_diff = calculate_log_prob_diff(actor_probs, rollout_probs, response_mask_bool)
+    rollout_probs_diff = calculate_diff(actor_probs, rollout_probs, response_mask_bool)
```
[the remark this answers is not in the document]
Comment (victordion) [reply verl/utils/debug/metrics.py:114]: This PR recognizes the current logic which is reporting on probs inseread of log probs.
